File: Symbols/Symbols.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;


namespace Sololearn
{
    class Program
    {
public static string re(string str)
{
int strz = str.Length;
string res="";
foreach(char x in str)
if(Char.IsLetter(x) || Char.IsDigit(x) || x == ' ')
res += x;
return res;
}
        static void Main(string[] args)
        {
      string msg = Console.ReadLine();
      Console.Write(re(msg));
        }
    }
}

File: Symbols/Symbols.java
import java.util.Scanner;

public class Program
{
public static String re(String str)
{
int strz = str.length();
String res = "";
for(int x = 0; x < strz;x++)
if(Character.isLetter(str.charAt(x)) || Character.isDigit(str.charAt(x)) || str.charAt(x) == ' ')
res += str.charAt(x);

return res;
}
 public static void main(String[] args)
{
Scanner input = new Scanner(System.in);
String msg = input.nextLine();
System.out.print(re(msg));
}
}
